php/libraries/NDB_Page.class.inc

```diff
@@ -601,6 +601,7 @@ class NDB_Page
                   $baseurl . "/js/util/queryString.js",
                   $baseurl . '/js/components/Form.js',
                   $baseurl . '/js/components/Markdown.js',
+                  $baseurl . '/js/components/Tabs.js',
```

Comment:
This isn't needed (`Tabs` are being imported via Webpack, so we don't need this global version). Also, this file doesn't even exist anymore and this will just cause 404s.